conf/etc/v2/alert_event_rules.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": "2.0.0",
```

Comment:
why do we need the version ?
Its not getting used anywhere in the code

Reply:
it will help during upgrade